packages/npm-bcrypt/wrapper.js

```diff
@@ -0,0 +1,38 @@
+var assert = require("assert");
+
+// The bcryptjs package has a slightly larger API than the native bcrypt
+// package, so we stick to the smaller API for consistency.
+var methods = {
+  compare: true,
+  compareSync: true,
+  genSalt: true,
+  genSaltSync: true,
+  getRounds: true,
+  hash: true,
+  hashSync: true
+};
+
+try {
+  // If you really need the native `bcrypt` package, then you should
+  // `meteor npm install --save bcrypt` into the node_modules directory in
+  // the root of your application.
+  var bcrypt = require("bcrypt");
+} catch (e) {
+  bcrypt = require("bcryptjs");
+  console.warn([
```

Comment:
That's great - I think it's acceptable to be noisy in this case as the users who typically care about noisiness are also the ones who will be eager to avoid using `bcryptjs`